app/controllers/reports/housed_addresses_controller.rb

```diff
@@ -72,7 +77,8 @@ def column_types
         'Last Name' => { query: c_t[:last_name], display_type: :text },
         'Housing Address' => { query: md_b_t[:address], display_type: :address },
         _('Did you or this client use external software to help with housing?') => { query: md_b_t[:external_software_used], display_type: :boolean },
-        'Date Housed' => { query: md_b_t[:client_move_in_date], display_type: :text },
```

Comment:
I renamed the "Date Housed" column to "Lease Start Date" for consistency, because that's what we call it everywhere else.

Reply:
Thanks!